taskwhisperer-extension@infinicode.de/extension.js

```diff
@@ -87,10 +87,6 @@ const ProjectHeaderBar = GObject.registerClass(class ProjectHeaderBar extends Po
     _init(menu) {
         this.menu = menu;
 
-        //this.actor.add(this._createLeftBoxMenu(), {expand: true, x_fill: true, x_align: St.Align.START});
```

Comment:
Removed, because `add` methods are deprecated. This code will not work with gnome 3.38.